na-mainloop/src/lib.rs

```diff
@@ -38,6 +39,14 @@ fn android_main(app: AndroidApp) {
                             }
                             MainEvent::InitWindow { .. } => {
                                 native_window = app.native_window();
+                                if let Some(nw) = &native_window {
+                                    nw.set_buffers_geometry(
+                                        0,
+                                        0,
+                                        Some(HardwareBufferFormat::R8G8B8A8_UNORM),
+                                    )
+                                    .unwrap()
+                                }
```

Comment:
It could be good to have a comment explaining what this is for.

This is intended to be a bare-bones example of how to use android-activity so it's quite likely someone looking at this isn't going to be familiar with what `set_buffers_geometry` does (and the name from the NDK isn't very self documenting when we're using it to set the format and not the 'geometry' as such).

Maybe we can just note that, since this example is going to clear the window from the CPU then we want to ensure the buffer is in a known format that we will be able to write to from `dummy_render()`

Reply:
Sure, still a draft as I wanted to share progress and show what's possible with the new bindings.

Reply:
Cool cool, this already shows a gradient instead of a black screen, as a sign of life.

We can probably drop the draft status now, IIRC this is mostly ready.

Reply:
Yeah don't mind if it's taken out of draft and will just want to wait until it doesn't depend on `[patches]` before landing.

Reply:
Sure, expect that to take ±1month though. That should be enough time to see if there are any issues with this patch though, pretty helpful to test compatibility with the ecosystem meanwhile.